shoop/themes/classic_gray/templates/classic_gray/shoop/front/product/category.jinja

```diff
@@ -88,6 +81,8 @@
 
 {% block extrajs %}
     <script>
+        // Absolute url for products reload
+        var ABSOLUTE_URL = "{{ request.path }}";
```

Comment:
No need for a separate variable. Use [`location.pathname`](https://developer.mozilla.org/en-US/docs/Web/API/Location) instead.